frontend/src/components/WofInfoBox.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <wof-popper>
+    <div :class="`flex-container flex-container--${this.type}`">
```

Comment:
Use computed property to set type for this.
For example: infoBoxType which will return proper class.